test/gtest/uct/test_amo_fadd_fxor.cc

```diff
@@ -60,6 +60,25 @@ UCS_TEST_SKIP_COND_P(uct_amo_fadd_fxor_test, fxor64,
     test_fop<uint64_t, UCT_ATOMIC_OP_XOR>(xor_op<uint64_t>);
 }
 
+UCS_TEST_SKIP_COND_P(uct_amo_fadd_fxor_test, dm_atomic_key_ksm_reg,
```

Comment:
the test name seems weird, it should not contain the words "ksm" or "dm" , it should be expressed by the UCT API terminology